xAPI.md

```diff
@@ -2483,8 +2483,19 @@ See <a href="#AppendixE">Appendix E: Example Signed Statement</a> for an example
 
 ## 3.0 Metadata
 
+Metadata is additional information about the resource of which it is associated.  It enables decision making,  
+search, and discoverability within Systems.  In xAPI, Metadata can take on many forms.  The most common of these 
+forms is within <a name="actdef"> Activity Definitions </a>, where the metadata describes the Activity. Most 
+types of metadata have the opportunity to have Extensions.  Other forms include Verbs, Activity types, and 
```

Comment:
Other forms of what? Metadata? A verb or activity type isn't a form of metadata, so I'm not sure how this is intended to be parsed